### ISSUE TITLE
Image Discovery App references external resources

### ISSUE THREAD

**Issue by ACueva** [Esri/ops-server-config#1367]
https://servername/SolutionsWeb/Apps/NationalIntel/ImageDiscovery/

Web application fails to load due to not being able to hit:

- [ ] dojo.css
- [ ] claro.css
- [ ] esri.css 
- [ ] ResizeHandle.css
- [ ] init.js

**Comment by elinz** [comment]
References 3.14, 3.13, and 3.11

**Comment by elinz** [comment]
Modified the env.js file to reference the local js api (i.e. /arcgis/jsapi/jsapicompact). Closing issue.

**Comment by elinz** [comment]
Re-opening issue. Upon testing the app on our development server after modifying the app to reference a local copy of the js api, I found the app does not load. It fails with the error "Unable to create map: a.trim is not a function".

Using Chrome developer tools it throws the following errors in the following files:
TypeError: a.trim is not a function       portalUtils.js:73
TypeError: a.trim is not a function       ConfigManager.js:118

I've tested the same app on an Ops Server deployment using 10.3.1 final software and the app does work using the local js api, so this indicates that the version of js api on final is newer (3.13) then the version of the js api on the development server which is a build of 10.3.1 very close to final (maybe one or two weeks at the most from final).

Short term solution: I'm rolling back the change I made to the env.js to reference the external js api (i.e. //js.arcgis.com/3.14) so we have a working app. Moving forward we plan to upgrade the development server to 10.4 software so this issue will be re-evaluated at that time.
